### DISCUSSION TITLE
2022-06-29 Weekly Meeting 62 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#248] [Weekly Meetings]
Weekly Meeting 62 is happening on Wednesday June 29th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

### Discussions: 
- Our role with other groups like MSF and do we want to participate as a group in MSF? 
- A new approach to how we ideate around problems, and demo findings! Making it more fun, and more about exploration than the past :)

**Comment by mrmetaverse** [comment]
A number of news journalists are reaching out regarding us and the new standards forum. We should continue our conversation on Wednesday about how we want to engage with other groups. We benefit from collaboration and it's not a us vs "them" scenario. OMIs work matters and were part of something larger. How we Collab with other groups will be a real turning point for Interoperability.

Furthermore, I think we can really shake things up with an experiment. I'd like to see us solve for challenge statements, and compare results. More demo days, less meeting minutes. More experimentation. More fun.

I'd like to use this week's time to have these two discussions. (this is a copy paste from #247)

**Comment by mrmetaverse** [comment]
# Media 
- would like to upgrade tweetshift to premium features 
  - We already have an integration running to our discord channel. So when our community syndicates tweets with our discord the premium features would include a button that enables discord users to click a button to retweet  / engage (direct from the channel)
  - This speeds up our workflow and helps us share stories together!
  - we are limited in terms of resources so automation can help alleviate some of our burden without sacrificing the integrity/consistency of our messaging. 
  - we should also look inward at our processes at ways to speed up decisions related to requests to test premium features from vendors, since (admittedly) this is the third week that media has asked for this. 
- Also seeking feedback related to tweetshift integration as-is. Visit the OMI-Media section of our discord, and engage.

**Reply by mrmetaverse** [reply]
- ~~lets take to a vote either next week or between now and next week.~~ its only $2 for a month trial... 
- propose timeframe of subscription for trial (we are going to try for 1 month/1 quarter/ etc.) 
- include price

**Comment by mrmetaverse** [comment]
# Community Egagement
- How do we want to engage with other groups? 
- How can we apply this to our stance on officially participating as a group in MSF? 
  - Some of these larger groups require information that  not everyone is comfortable sharing ("real name", address.. etc.) 
  - We may be overthinking it, and we could reach out to Neil and ask directly - what are the actual tangible requirements? "We could just find that out by asking" - humbletim
  - A member proposed voting to see if we are interested 
  - A member proposed we reach out and ask our questions directly. 
  - "I think it would be great to have OMI logo on the MSF page too, showing our support for the effort" 
  - issue: You must have your org email to participate as a member of OMI. 
    - do we implement OMI emails? 
    - does OMI register with admin@omigroup? and then individuals 
  - " I think that we belong at the table, and already have a voice. We have communal capital, and have been doing this together with and without large companies!" 
 
 Decisions: 
 - Does OMI participate as a group in the first July forum? 
 - Does OMI submit our logo as a group? 
 - If individual members of OMI want to register as themselves using OMI as their org, can we get them each gmail accounts if needed?

**Reply by mrmetaverse** [reply]
Does OMI participate as a group in the first July forum?
- most seem interested, but we will allow each other to decide on their own.

**Reply by mrmetaverse** [reply]
Does OMI submit our logo as a group?

we will create an async vote

**Reply by mrmetaverse** [reply]
If individual members of OMI want to register as themselves using OMI as their org, what does that require and look like?

- Q+A session so our members can ask the questions they need to know (identity, address, intent, etc.)
- can we get each (a) gmail account(s) if needed?

**Comment by mrmetaverse** [comment]
## Shaking things up: A culture of experimentation and more demos/talks
>Furthermore, I think we can really shake things up with an experiment. I'd like to see us solve for challenge statements, and compare results. More demo days, less meeting minutes. More experimentation. More fun.

### Group thoughts 
- "seems like every week we have someone demoing something amazing" 
  - "rally behind recognizing that and aiming for that"
- Our group decided it would be fun to "do more stuff" 
- A lot of our members have topics and ideas to share already. We could pick a theme each week, or two weeks, and share our experiments and findings. 
- Demos of all kind "not always code!" 
- We could share our findings more often. 

### So what is a good first problem to solve for and experiment? 
- `Avatars`: Next Wednesday lets talk about Avatars

#### Future
- `Portals`: How should they work? What have you experimented with? What has your company been doing lately for portals? Codefrau (Croquet) said "happy to do a deep-dive into the portals I built". 
- `Homespaces` What could interoperable homespace templating look like? Jin suggested doing workshops, and discussions on VTubing, and more. 
- `content portability` 

![image](https://user-images.githubusercontent.com/63426722/176493588-91820c77-1d37-4e37-bc35-868f57817708.png)
![image](https://user-images.githubusercontent.com/63426722/176493644-943821fc-acdf-4da7-be68-f4beb831ef8a.png)

**Comment by mrmetaverse** [comment]
# Next week we will discuss `Avatar Interoperability`. Its a broad category so bring your findings, your musings, and your suggestions for others!

**Reply by mrmetaverse** [reply]
## After thoughts
- Media team to swarm announcing next weeks theme, and during next weeks demos we will tweet out screenshots and shoutouts of the demos. 
- "Furthering our commitment to the open Metaverse" future blog post for MSF participation. 
- posting items for vote in the voting channel 
  - do we submit our logo now to MSF in public support of the effort as OMI? This is creating OMI as an org in their system.

**Reply by pcsmgnt** [reply]
Polls posted on Discord for two days to allow people the time and the chance to vote. Will check response ahead of next week's meeting.
